middleware.go

```diff
@@ -101,7 +102,8 @@ func Session(cfg Options) func(next http.Handler) http.Handler {
 				serviceClaim, _ := claims["service"].(string)
 				accountClaim, _ := claims["account"].(string)
 				adminClaim, _ := claims["admin"].(bool)
-				projectClaim, _ := claims["project"].(float64)
+				// TODO: we support both claims for now, we'll eventually deprecate one.
```

Comment:
I think let's remove TODO and explain what service set `project_id` instead of `project` and what's exactly affected (secret keys from builder?)